scripts/background.js

```diff
@@ -1,3 +1,79 @@
+function clearBadge() {
```

Comment:
could we split all the notification code into a separate file? let's call it `notification.js`